requirements/ci_requirements.txt

```
@@ -1,4 +1,4 @@
 spin==0.13
 # Keep this in sync with ci32_requirements.txt
-scipy-openblas32==0.3.29.0.0
-scipy-openblas64==0.3.29.0.0
+scipy-openblas32==0.3.29.265.0
+scipy-openblas64==0.3.29.265.0
```

Hmmm, can't install this on arm windows yet, so I think something like:
```suggestion
scipy-openblas64==0.3.29.265.0 ; sys_platform != 'win32' or platform_machine != 'arm64'
```

---

whoops, yeah, makes sense